tests/test_cases/pipeonestring.py

```diff
@@ -0,0 +1,21 @@
+#!/bin/python2
+# Checks if pipeonestring.c runs the same natively as it does within Lind
+import sys
+
+lind_result = sys.argv[1].split('\n')
+host_result = sys.argv[2].split('\n')
+
+if len(lind_result) != len(host_result):
+    print "Mismatched number of lines!"
+    exit(-1)
+
+for line in lind_result:
+    if line[0] == 'I' and (not line.split(": ")[1].isdigit()):
+        print "Nondeterministic lines of an unrecognized format!"
+        exit(-1)
+
+for line in host_result:
+    if line[0] == 'I' and (not line.split(": ")[1].isdigit()):
+        print "Nondeterministic lines of an unrecognized format!"
+        exit(-1)
+        
```

Comment:
The more precise the testing we can do the better, so if you can check the first n characters rather than the first one, I'd opt for doing that.

Reply:
This goes for basically any test

Reply:
What you should do to check this is check that the host and lind output match up until that digit (on the other side of that split.

Reply:
Also just checking deterministic lines for equality is good for completeness